pkgs/servers/jellyfin/default.nix

```diff
@@ -55,4 +55,8 @@ in stdenv.mkDerivation rec {
     license = licenses.gpl2;
     maintainers = with maintainers; [ nyanloutre minijackson ];
   };
+
+  passthru.tests = {
+    inherit (nixosTests) jellyfin;
+  };
```

Comment:
can we order the passthru attribute above `meta`? Just an aesthetic preference.
If we `nix-build` this it looks a little funny
```
nix-build -A jellyfin.tests.jellyfin
```

could you do something like
```
{
  what-this-test-does = jellyfin;
};
```

just for a bit more information about the test case?